ramls/users.raml

```diff
@@ -51,3 +51,17 @@ resourceTypes:
         schema: user
     get:
       description: Get a single user
+  /expire/timer:
+    post:
+      description: Expire timer (timer event)
+      responses:
+        204:
+          description: Expired ok
+        400:
+          description: User Error
+          body:
+            text/plain:
```

Comment:
Is this needed?  There is no implementation to return 400.

Reply:
RMB may return it - before our handler is called.